lib/tasks/data/categories.yml

```diff
@@ -0,0 +1,208 @@
+---
+  A:
+    title: Category A
+    move_supported: false
+    locations:
+  B:
+    title: Category B
+    move_supported: true
+    locations:
+    - FKI
+    - FNI
+    - GHI
+    - GTI
+    - IWI
+    - LLI
+    - LGI
+    - RHI
+    - SLI
+    - WDI
+    - WRI
+    - MRI
+    - WHI
+  BTH:
+    title: Category B Therapeutic
+    move_supported: true
+    locations:
+    - GNI
+  C:
+    title: Category C
+    move_supported: true
+    locations:
+    - HCI
+    - MSI
+    - CWI
+    - BXI
+    - BRI
+    - HPI
+    - ISI
+    - LTI
+    - MTI
+    - WII
+    - WLI
+    - FSI
+    - HMI
+    - LHI
+    - MDI
+    - OWI
+    - ONI
+    - RNI
+    - SFI
+    - SKI
+    - SNI
+    - WEI
+    - BCI
+    - HHI
+    - LFI
+    - NLI
+    - RSI
+    - WMI
+    - CLI
+    - DAI
+    - EEI
+    - GMI
+  CVPU:
+    title: Category C VPU
+    move_supported: true
+    locations:
+    - WTI
+    - ASI
+    - VEI
+    - UKI
+  CYOI:
+    title: Category C / YOI
+    move_supported: true
+    locations:
+    - SHI
+    - HII
+    - PDI
+    - RCI
+  D:
+    title: Category D
+    move_supported: true
+    locations:
+    - HBI
+    - HDI
+    - NSI
+    - SUI
+    - HVI
+    - KMI
+    - KVI
+    - TCI
+    - FDI
+    - LYI
+    - SPI
+    - EHI
+    - UPI
+  E:
+    title: Category A Ex
+    move_supported: false
+    locations:
+  EL:
+    title: Eligible
+    move_supported: true
+    locations:
+  FEM:
+    title: Female
+    move_supported: true
+    locations:
+    - BZI
+    - DWI
+    - DHI
+    - EWI
+    - FHI
+    - LNI
+    - NHI
+    - PFI
+    - SDI
+    - STI
+  FEMO:
+    title: Female Open
+    move_supported: true
+    locations:
+    - ESI
+    - AGI
+  GRANTED:
+    title: Parole Granted
+    move_supported: true
+    locations:
+  H:
+    title: Cat A Hi
+    move_supported: false
+    locations:
+  HI:
+    title: High
+    move_supported: true
```

Comment:
lets confirm with DPS that `HI` moves should be supported

Reply:
Okay. I think I'll merge this as is for now and then amend the yaml file if needed as that will be a small tweak and I need this merged to unblock the next story.